apps/checkout/src/OrderConfirmation.tsx

```diff
@@ -0,0 +1,39 @@
+import { Suspense } from "react";
+
+import PageHeader from "@/sections/PageHeader";
+import { FinalizedSummary } from "@/sections/Summary";
+import { OrderInfo } from "@/sections/OrderInfo";
+import { SummaryPlaceholder } from "@/sections/Summary/SummaryPlaceholder";
+import { Title } from "@/components/Title";
+import { Text } from "@/components/Text";
+import { useOrder } from "@/hooks/useOrder";
+import { useFormattedMessages } from "@/hooks/useFormattedMessages";
+
+export const OrderConfirmation = ({ orderToken }: { orderToken: string }) => {
+  const { order } = useOrder(orderToken);
+  const formatMessage = useFormattedMessages();
+
+  return (
+    <div className="page">
+      <header className="order-header">
+        <PageHeader />
+        <Title>
+          {formatMessage("orderConfirmationTitle", { number: order.number })}
+        </Title>
+        <Text size="md" className="max-w-[692px]">
```

Comment:
Do we really want to define width in such a way, not in some global config?

Reply:
it's a one-off setting, I would not make it a variable